descriptors/ModuleDescriptor-template.json

```diff
@@ -1,6 +1,6 @@
 {
   "id": "@artifactId@-@version@",
-  "name": "The module descriptor for mod-fqm-manager.",
+  "name": "FQM Manager Module",
```

Comment:
❤️ 
This drives me nuts every time I look at the list of installed modules in any environment 🙂